src/command/commands/ps.rs

```diff
@@ -117,10 +117,10 @@ impl CommandHandler for Ps {
         request
     }
 
-    fn handle(&mut self, payload: &[u8]) -> Option<CoapRequest<String>> {
-        self.payload = payload.to_vec();
+    fn handle(&mut self, response: &Packet) -> Option<CoapRequest<String>> {
+        self.payload = response.payload.to_vec();
```

Comment:
Clippy / CI is unhappy:
consider using: `response.payload.clone()`

Reply:
just force push. Fix ups only really make sense if you expect multiple rounds of reviews I think.